app/src/main/java/com/mediaan/mymediaan/view/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,137 @@
+package com.mediaan.mymediaan.view.profile
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.DrawerState
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.Green
+import androidx.compose.ui.graphics.Color.Companion.Red
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.mediaan.mymediaan.model.TwoTruthsOneLieEntity
+import com.mediaan.mymediaan.repository.ProfileRepository
+import com.mediaan.mymediaan.ui.theme.MediaanPrimary
+import com.mediaan.mymediaan.ui.theme.Typography
+import com.mediaan.mymediaan.view.MyMediaanAppBar
+
+@Composable
+fun ProfileScreen(
+    drawerState: DrawerState,
+    profileId: String?,
+    profileRepository: ProfileRepository = ProfileRepository()
+){
+    val profile = remember(profileId) {
+        profileId?.let { profileRepository.getProfileById(it) }
+    }
+    val scrollState = rememberScrollState()
+
+    Scaffold(topBar = {
+        MyMediaanAppBar(
+            drawerState = drawerState,
+            title = "Profile",
```

Comment:
Don't you wanna use `stringResource(MyMediaanScreen.Profile.title)` here instead of hardcoded value?